### PR TITLE
Copy Pytorch sample into solution

### PR DESCRIPTION
**Pull request by asvrada**
Copied `samples/pytorch` into `solution/pytorch_inference`

This test would takes 10+ minutes to run (make && make run)

### CODE REVIEW

**Comment by vtikoo** [comment]
> This test would takes 10+ minutes to run (make && make run)

This moves the pr-pipeline running time very close to the 60 min timeout.

Unless this is a scenario we care a lot about, we should move it to the nightly pipeline.

**Comment by asvrada** [comment]
Previously pipeline would only take ~45mins to run, this single solution test indeed increases the time by a lot. Will move this test case to the nightly pipeline.

File: solutions/pytorch_inference/misc/torch_nn.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    print("Done")
+
+def main():
+    mp.set_start_method('spawn')
+    train()
+
+if __name__ == "__main__":
+    main()
+
```
Comment (vtikoo) [inline solutions/pytorch_inference/misc/torch_nn.py:66]: formatting nit: Remove additional trailing line.

**Review by vtikoo — APPROVED** [review]
Looks good! Added a minor comment about formatting.

File: solutions/pytorch_inference/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ROOTFS        = ext2rootfs
+TIMEOUT = 180
+
+QUERY_INTERVAL = 2
+SHUTDOWN_WAIT = 2 
+
+all: appdir build
+
+appdir:
+	$(APPBUILDER) -v -d Dockerfile.rootfs
```
Comment (bodzhang) [inline solutions/pytorch_inference/Makefile:18]: `-v` generates a lot of docker build log. In pipeline run, might be wasting storage, without much value

File: solutions/pytorch_inference/misc/torch_polynomial_nn.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Tutorial code from https://pytorch.org/tutorials/beginner/examples_nn/polynomial_nn.html
```
Comment (bodzhang) [inline solutions/pytorch_inference/misc/torch_polynomial_nn.py:1]: This file is not used.

File: .azure_pipelines/ci-pipeline-makefile.yml [hunk 4]
```diff
@@ -75,6 +75,8 @@ jobs:
           DB_NAME: $(DB_NAME_USEAST)
           DB_SERVER_NAME: $(DB_SERVER_NAME_USEAST)
           MAA_URL: $(DB_MAA_URL_USEAST)
+          # If set, don't run certain tests
+          MYST_SKIP_PR_TEST: 1
```
Comment (bodzhang) [inline .azure_pipelines/ci-pipeline-makefile.yml:79]: Is this configuration only applied in PR pipeline? The name of the configuration is confusing.

**Pull request merged**